command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
I think this should be a part of the runtime spec. Without this CLI, there will be no compliance testing or interoperability between runtimes at all.